docs/configuration.rst

```diff
@@ -4,7 +4,8 @@ Configuration
 =============
 
 PyWPS is configured using a configuration file. The file uses the
-`ConfigParser <https://wiki.python.org/moin/ConfigParserExamples>`_ format.
+`ConfigParser <https://wiki.python.org/moin/ConfigParserExamples>`_ format, with
+interpolation initialised using `os.osenviron`.
```

Comment:
I think, this is wrong `os.osenviron` -> `os.environ` ?

Reply:
ya, merci!